recipes/expat/all/conandata.yml

```diff
@@ -11,3 +11,11 @@ sources:
   "2.2.10":
     sha256: bf42d1f52371d23684de36cc6d2f0f1acd02de264d1105bdc17792bbeb7e7ceb
     url: https://github.com/libexpat/libexpat/releases/download/R_2_2_10/expat-2.2.10.tar.gz
+  "2.3.0":
+    sha256: 89df123c62f2c2e2b235692d9fe76def6a9ab03dbe95835345bf412726eb1987
+    url: https://github.com/libexpat/libexpat/releases/download/R_2_3_0/expat-2.3.0.tar.gz
+patches:
+  2.3.0:
+    - patch_file: "patches/relax-vs-restriction.patch"
+      base_path: "source_subfolder"
+      strip: 2
```

Comment:
```suggestion

```

Reply:
I left the patch as it was generated by `git diff` and use `strip: 2` to remove `a/expat/` from the paths.
I find this is much easier to maintain than manually editing patches, as there are generally multiple files being patched.  For that reason, I'm a big fan of `strip: N` for patches.

Reply:
hooks don't allow `strip` keyword in conandata.yml, and I see that your patch still contains `a/expat`